sql/src/main/java/io/crate/expression/symbol/Literal.java

```diff
@@ -276,6 +277,10 @@ public void writeTo(StreamOutput out) throws IOException {
         return new Literal<>(DataTypes.FLOAT, value);
     }
 
+    public static Literal<MonthDaySecondInterval> newInterval(MonthDaySecondInterval MOnthDaySecondInterval) {
+        return new Literal<>(DataTypes.INTERVAL, DataTypes.INTERVAL.value(MOnthDaySecondInterval));
```

Comment:
Given that the argument is already restricted to `MonthDaySecondInterval` there is no need to call `INTERVAL.value`.